pkg/systemd/services.html

```diff
@@ -153,37 +153,37 @@ <h1 translate>No Matching Results</h1>
           <i class="fa fa-caret-down pf-c-context-selector__toggle-icon" aria-hidden="true"></i>
         </button>
         <ul class="dropdown-menu vertical-scroll">
-          <li value="1"><a tabindex="0" translate>1st</a></li>
-          <li value="2"><a tabindex="0" translate>2nd</a></li>
-          <li value="3"><a tabindex="0" translate>3rd</a></li>
-          <li value="4"><a tabindex="0" translate>4th</a></li>
-          <li value="5"><a tabindex="0" translate>5th</a></li>
-          <li value="6"><a tabindex="0" translate>6th</a></li>
-          <li value="7"><a tabindex="0" translate>7th</a></li>
-          <li value="8"><a tabindex="0" translate>8th</a></li>
-          <li value="9"><a tabindex="0" translate>9th</a></li>
-          <li value="10"><a tabindex="0" translate>10th</a></li>
-          <li value="11"><a tabindex="0" translate>11th</a></li>
-          <li value="12"><a tabindex="0" translate>12th</a></li>
-          <li value="13"><a tabindex="0" translate>13th</a></li>
-          <li value="14"><a tabindex="0" translate>14th</a></li>
-          <li value="15"><a tabindex="0" translate>15th</a></li>
-          <li value="16"><a tabindex="0" translate>16th</a></li>
-          <li value="17"><a tabindex="0" translate>17th</a></li>
-          <li value="18"><a tabindex="0" translate>18th</a></li>
-          <li value="19"><a tabindex="0" translate>19th</a></li>
-          <li value="20"><a tabindex="0" translate>20th</a></li>
-          <li value="21"><a tabindex="0" translate>21st</a></li>
-          <li value="22"><a tabindex="0" translate>22nd</a></li>
-          <li value="23"><a tabindex="0" translate>23rd</a></li>
-          <li value="24"><a tabindex="0" translate>24th</a></li>
-          <li value="25"><a tabindex="0" translate>25th</a></li>
-          <li value="26"><a tabindex="0" translate>26th</a></li>
-          <li value="27"><a tabindex="0" translate>27th</a></li>
-          <li value="28"><a tabindex="0" translate>28th</a></li>
-          <li value="29"><a tabindex="0" translate>29th</a></li>
-          <li value="30"><a tabindex="0" translate>30th</a></li>
-          <li value="31"><a tabindex="0" translate>31st</a></li>
+          <li value="1"><a role="link" tabindex="0" translate>1st</a></li>
```

Comment:
I wonder if the UL above should be role=menu and these should be role=menuitem instead?

Reply:
Possibly. But by the same logic we should fix it in may other places (even in this PR). I suggest doing that in another PR.

Reply:
But even better would be to replace this by `<select>`